vanilla_installer/gtk/widget-partition.ui

```diff
@@ -120,7 +120,7 @@
     <child>
       <object class="AdwPreferencesGroup" id="var_part">
         <property name="title" translatable="yes">Var Partition</property>
-        <property name="description" translatable="yes">The partition to mount under “/var”. This directory stores all user data like personal documents and system-wide configuration.</property>
+        <property name="description" translatable="yes">The partition to mount under “/var”. This directory stores all user data like personal documents and system-wide configuration. Minimum space required in the partition is 5GiB.</property>
```

Comment:
I don't think the installer checks for the size of /var.
5GB is too little tho, should be somewhere around 25-30 GB